README.md

```diff
@@ -252,6 +252,7 @@ The supported rules are described below:
 | no_response_codes         | Flag any response object that has no valid response codes.   | oas3 |
 | no_success_response_codes | Flag any response object that has no success response codes. | oas3 |
 | no_response_body          | Flag any non-204 success responses without a response body.  | oas3 |
+| ibm_status_code_guidelines| Flag any violations of status code conventions per IBM API Handbook  | oas3 |
```

Comment:
I like the idea of having these rules wrapped under a general key to avoid too much verbosity between similar rules (especially for rules specific to IBM). That said, it's a slight deviation from previous patterns so I'd like to see what @mkistler thinks as well

Reply:
I wish I could take credit, but I stole the idea from @mkistler!